package.json

```diff
@@ -178,6 +178,7 @@
 		"fixtures:generate": "npm run fixtures:server-registered && cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit",
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
 		"lint": "concurrently \"npm run lint-js\" \"npm run lint-pkg-json\" \"npm run lint-css\" \"npm run lint-types\"",
+		"lint-es5-only": "npx eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore ./build/**/*.js",
```

Comment:
I think we should inline this into the Travis config. It's not really something developers will run locally and `npx` and `eslint` are not direct, local dependencies.

Reply:
Hmm good point. Although in theory devs could use it locally to check if new dependencies are IE-compatible, it'll probably never happen 😁

Reply:
Noting two things:

- `npx` here would be redundant anyways, since NPM will [include `./node_modules/.bin` in the `PATH`](https://docs.npmjs.com/cli/run-script) when running any script. So the line here would work fine (the same) if running `eslint` directly.
- If ESLint is not a defined dependency, then we shouldn't have confidence that it would be installed in Travis either. I have a feeling this is only incidentally working because it's some transitive dependency of another package (probably one of the ESLint plugins which _are_ defined as dependencies).

Reply:
But `npx` installs it if it's not there.

Reply:
> But `npx` installs it if it's not there.

Ah, you're right! I always forget that part of it.